conda/exception_handler.py

```diff
@@ -239,147 +237,6 @@ def _isatty(self):
             log.debug("%r", e)
             return True
 
-    def _upload(self, error_report) -> None:
```

Comment:
Instead of removing this right away, we'll want to make this go through the deprecation cycle.

```
@deprecated(
        "26.9",
        "27.3",
        addendum="`_upload` is being deprecated and will be removed in conda version 27.3",
    )
```

Reply:
same for `_ask_upload`,`_execute_upload` and `_post_upload`